WordPress/Classes/ViewRelated/NUX/SiteCreationDomainsActivityTableViewCell.xib

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is throwing a warning: `Ambiguous Layout: Vertical position is ambiguous for "Activity Spinner".`